crates/bindings-csharp/Runtime/Internal/Module.cs

```diff
@@ -233,29 +233,29 @@ private static byte[] Consume(this BytesSource source)
             // Write into the spare capacity of the buffer.
             var spare = buffer.AsSpan((int)written);
             var buf_len = (uint)spare.Length;
-            var ret = FFI._bytes_source_read(source, spare, ref buf_len);
+            var ret = (FFI.Errno)FFI._bytes_source_read(source, spare, ref buf_len);
```

Comment:
I'd change `bytes_source_read` to return `Errno` directly instead, to avoid casting.